.github/workflows/build-pkg.yml

```diff
@@ -24,16 +27,51 @@ jobs:
         id: print-version
         run: |
           echo "version=$(git -C brew describe --tags --always)" > $GITHUB_OUTPUT
+      - name: Install the Apple certificate
+        env:
+          PKG_BUILD_CERTIFICATE_BASE64: ${{ secrets.PKG_BUILD_CERTIFICATE_BASE64 }}
+          PKG_BUILD_P12_PASSWORD: ${{ secrets.PKG_BUILD_P12_PASSWORD }}
+          KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
```

Comment:
Maybe we can just do `uuidgen` in the run block below to generate a UUID as a password for this?